### DISCUSSION TITLE
Gaming

### DISCUSSION THREAD

**Discussion by shandshellin** [DustinBrett/daedalOS#380] [Feature Requests]
Love the project!

How can i full screen quake 3? Is it possible to add custom dos game to this image or maybe bind a volume to it to load from outside?

**Comment by DustinBrett** [comment]
Thanks! I recently added maximize support to Quake 3, but in general I do not yet have support for most apps in full screen. It's something I will consider adding, like I have done for Photos. Thanks for the idea to add it.

Indeed you can mount external folders from the local system to load custom dos games. You can also drag/drop files in. And on my website I have my favorite 13 DOS Shareware games.

https://dustinbrett.com/?url=/Users/Public/Documents/Games/DOS%20Bundles

**Comment by shandshellin** [comment]
Is there an exisitng folde i can map to or if use a new naem will it create it for me in the os when it mounts?

**Comment by DustinBrett** [comment]
> Is there an exisitng folde i can map to or if use a new naem will it create it for me in the os when it mounts?

No there is nothing pre-existing and the mounting feature is just simply connecting a local folder into the site.
